web/js/components/layer/product-picker/browse/browse-layers.js

```diff
@@ -263,9 +269,11 @@ function mapStateToProps(state) {
   return {
     browser,
     mode,
+    selectedCategoryName: category && category.title,
     categoryType,
     categoryTabNames: config.categoryGroupOrder,
     measurementConfig: config.measurements,
+    lastAction,
```

Comment:
Shouldn't need `lastAction` anymore

Reply:
Addressed in this commit: https://github.com/nasa-gibs/worldview/pull/3783/commits/524bffbe94b6f200d389a78034f31ed7e6bfcfec